.github/workflows/deploy-staging.yml

```diff
@@ -0,0 +1,81 @@
+name: deploy to staging
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  format:
+    name: 'Check code formatting'
+    runs-on: ubuntu-20.04
+    defaults:
+      run:
+        shell: bash -l {0}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          activate-environment: <dashboard name> # Update name to match dashboard.
+          environment-file: environment.workspace.yml
```

Comment:
I think this should be `environment.yml`.  The `environment.workspace.yml` installs the packages within the DAP, and the `environment.yml` is for staging.

Ideally we would only have 1 file, which is a ticket in the TODO.